plugins/processors/awsapplicationsignals/internal/resolver/kubernetes.go

```diff
@@ -78,47 +91,74 @@ func getKubernetesResolver(platformCode, clusterName string, logger *zap.Logger)
 		// jitter calls to the kubernetes api
 		jitterSleep(jitterKubernetesAPISeconds)
 
-		sharedInformerFactory := informers.NewSharedInformerFactory(clientset, 0)
-		podInformer := sharedInformerFactory.Core().V1().Pods().Informer()
-		err = podInformer.SetTransform(minimizePod)
-		if err != nil {
-			logger.Error("failed to minimize Pod objects", zap.Error(err))
-		}
-		serviceInformer := sharedInformerFactory.Core().V1().Services().Informer()
-		err = serviceInformer.SetTransform(minimizeService)
-		if err != nil {
-			logger.Error("failed to minimize Service objects", zap.Error(err))
-		}
+		useListPod := (os.Getenv("APP_SIGNALS_USE_LIST_POD") == "true")
```

Comment:
nit: Consider making this a const with a comment for its purpose and specifying that it's temporary. This seems like something that would be a good candidate for a feature gate.

Reply:
I will change it accordingly.